rust/src/combinators/measure_cast/zCDP_to_approxDP/cdp_delta/test.rs

```diff
@@ -0,0 +1,15 @@
+use super::*;
+#[test]
+fn test_edge_cases() -> Fallible<()> {
+    // negativity checks
+    assert!(cdp_delta(-0., 0.).is_err());
+    assert!(cdp_delta(0., -0.).is_err());
+
+    assert_eq!(cdp_delta(0., 0.)?, 0.);
+    assert_eq!(cdp_delta(0.5, 0.)?, 0.5588356393474351);
```

Comment:
ditto: replace with expression?

Reply:
The expression is pretty complex. I just copied the prior implementation and checked for equality. I think it would be good to keep the reverse map from bit rotting as well.

Reply:
I have also checked the new implementation against the reference implementation.